integration-test/1482-maritime_boundary-buffered_land.py

```diff
@@ -4,54 +4,6 @@
 # Adds tests for OSM features (but not NE features)
 class MaritimeBoundary(FixtureTest):
 
-    def test_usa_canada_country_boundary(self):
```

Comment:
Are these tests being replaced with something else? I see these IDs testing the `sort_key`, but not the `kind` later on in the test section?

Reply:
I couldn't tell what they were testing, other than the maritime / non-maritime status that was replicated in the generative tests below. Since, as part of adapting the tests for the new line / polygon status, I was converting stuff to generative testing anyway, it seemed unnecessary to convert these when generative equivalents already existed.